website/docs/subo.md

```diff
@@ -3,59 +3,70 @@ pagination_prev: null
 pagination_next: null
 ---
 
-# Install the Suborbital CLI
+# Subo: the Suborbital CLI
 
-Subo is the command-line tool for working with the Suborbital Extension Engine (SE2). Subo is used to build SE2 modules, generate new projects and config files, and more over time.
+![alt="panda in a spacesuit"](https://user-images.githubusercontent.com/5942370/129103528-8b013445-a8a2-44bb-8b39-65d912a66767.png)
+
+Subo is the command-line tool for working with the Suborbital Extension Engine (SE2). Subo is used to build SE2 modules, generate new projects and config files, and is also our mascot 🐼.
 
 **You do not need to install language-specific tools to get started with SE2 and Subo!** A Docker toolchain is supported (see below) that can build your SE2 modules without needing to install language toolchains.
 
-## Installing
+## Install Subo
+
+The `subo` tool supports the following platforms and operating systems:
+
+|         | x86_64 | arm64 |
+| ------- | ------ | ----- |
+| Mac     | ✅     | ✅    |
+| Linux   | ✅     | ✅    |
+| Windows | 🚫     | 🚫    |
+
+The language toolchains used by `subo` support the following platforms:
+
+|                | x86_64 | arm64 | Docker |
+| -------------- | ------ | ----- | ------ |
+| Rust           | ✅     | ✅    | ✅     |
+| AssemblyScript | ✅     | ✅    | ✅     |
```

Comment:
While we're here, is this still accurate / complete?

Reply:
Updated!